frontend/util/fetch.js

```diff
@@ -60,20 +60,26 @@ function fetchWrapper(url, configs = {}) {
 
   let requestConfigs;
 
+  let requestBody = JSON.stringify(configs.data);
+
+  if (baseConfigs.headers['content-type'] === 'multipart/form-data') {
+    requestBody = configs.body;
+    // browser must set this for FormData
+    delete baseConfigs.headers['content-type'];
```

Comment:
After an upload, do the following requests have the `content-type` set to `application/json`?

Reply:
they'd get `defaultHeaders` again which includes ` 'content-type': 'application/json'`, unless they have been explicitly given `'multipart/form-data'`